FF8.Core/Sources/FieldId.cs

```diff
@@ -0,0 +1,990 @@
+﻿using System;
+
+namespace FF8.Core
+{
+    public enum FieldId
```

Comment:
fields are read from "maplist" in field.fs- every field is tied to ID. 
See https://github.com/MaKiPL/OpenVIII/blob/master/FF8/init_debugger_fields.cs#L22